test/e2e-tests-auction.sh

```diff
@@ -30,6 +30,14 @@ step "Place 6 orders in current Auction" \
  truffle exec scripts/sell_order.js 4 1 0 4 52  && \
  truffle exec scripts/sell_order.js 5 2 0 20 280"
 
+step_with_retry "[theGraph]: SellOrder was added to graph db - accountId 5's sellOrder === 280" \
```

Comment:
nit: "[theGraph]:" looks kinda awkward next to all the other tests...

Reply:
will remove and retry build